snuba/settings_base.py

```diff
@@ -13,7 +13,10 @@
 DATASET_MODE = 'local'
 
 # Clickhouse Options
-CLICKHOUSE_SERVER = os.environ.get('CLICKHOUSE_SERVER', 'localhost:9000')
+# TODO: Warn about using `CLICKHOUSE_SERVER`, users should use the new settings instead.
+[default_clickhouse_host, default_clickhouse_port] = os.environ.get('CLICKHOUSE_SERVER', 'localhost:9000').split(':', 1)
+CLICKHOUSE_HOST = os.environ.get('CLICKHOUSE_HOST', default_clickhouse_host)
+CLICKHOUSE_NATIVE_PORT = int(os.environ.get('CLICKHOUSE_NATIVE_PORT', default_clickhouse_port))
```

Comment:
Same naming question here about `CLICKHOUSE_NATIVE_PORT` via `CLICKHOUSE_PORT` to disambiguate with `CLICKHOUSE_HTTP_PORT`.